emission/tests/analysisTests/TestSimilarity.py

```diff
@@ -86,10 +94,10 @@ def testDeleteBins(self):
             self.assertTrue(b == sim.num)
 
     def testElbowDistance(self):
-        start = Coordinate(47,-122)
-        end = Coordinate(47,-123)
-        now = datetime.datetime.now()
-        t = Trip(None, None, None, None, now, now, start, end)
+        start = [-122,47]
+        end = [-133,47]
```

Comment:
is the x coordinate a typo or did you change it compared to old test? (-123 instead of -133)